internal/builder/builder.go

```diff
@@ -1257,3 +1305,11 @@ func (e errModuleTar) Info() dist.ModuleInfo {
 func (e errModuleTar) Path() string {
 	return ""
 }
+
+func cnbBuildConfigDir() string {
```

Comment:
IIUC this allows the pack user to specify the location of the build config directory within the builder by setting an environment variable in pack's environment. We might want to add some text to `pack builder create --help` so that this option is discoverable. Or maybe we could add it as a field in builder.toml ...or just leave it non-configurable. WDYT @WYGIN @jjbustamante?

Reply:
i am not sure about adding `pack builder create --help`, IMO the `pack builder create --help` is about giving better context to user about the list of commands available for the end-user to use, and i think `CNB_BUILD_CONFIG_DIR` in cli help might not required
so i added it to the docs where [builder.toml](https://github.com/buildpacks/docs/pull/622) fields and tables are specified